yt/frontends/art/data_structures.py

```diff
@@ -320,7 +320,7 @@ def _parse_parameter_file(self):
             self.parameters["wspecies"] = wspecies[:n]
             self.parameters["lspecies"] = lspecies[:n]
             for specie in range(n):
-                self.particle_types.append("specie%i" % specie)
+                self.particle_types.append(f"specie{specie}")
```

Comment:
I believe in English, 'species' is actually invariable, but I'm keeping bug-for-bug compatibility in this pure refactor

Reply:
the [etymology rabbit hole for "specie"](https://en.wiktionary.org/wiki/specie#English) is kinda fun. I could be convinced that it's more appropriate to use specie for particle types in a numerical simulation: maybe having a variety of particle types is more similar to minting different types of coins than it is to variations in taxonomical classification. But most likely species was meant here :)